docs/platform/concepts/service_backups.rst

```diff
@@ -123,7 +127,7 @@ Aiven for Apache Cassandra backups are taken every 24 hours. The point-in-time r
 
 .. note::
     
-    If you'd like to be notified once the PITR feature is available for Cassandra, contact the Aiven support.
+    If you'd like to be notified once the PITR feature is available for Cassandra, contact Aiven support.
```

Comment:
I'd keep "the" here (for me - one specific group of individuals designated to support our customers). No need to discuss - just follow your intuition :-)